client/src/components/overview/AddToCart.jsx

```diff
@@ -1 +1,68 @@
-import React from 'react';
\ No newline at end of file
+import React, { useState } from 'react';
+import { Container, Button, Row, Form, FormGroup, FormControl, Col } from 'react-bootstrap';
+import { useSelector } from 'react-redux';
+import API_KEY from '../../config/config'
+import axios from 'axios';
+
+function AddToCart(props) {
+  const widget = 'add-to-cart'
+  const skusInfo = useSelector((state) => state.style.skus);
+  const skus = Object.keys(skusInfo);
+  const sizes = [];
+  Object.values(skusInfo).forEach(sku => {
+    sizes.push(sku.size);
+  });
+  const sizeSelector = sizes.map((size, index) => <option  key={index} value={skus[index]}>{size}</option>);
+  const [size, setSize] = useState('');
+  const [quantity, setQuantity] = useState(null);
+  const [atcQuantity, setATCQuantity] = useState(0);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (size === '' || atcQuantity < 1) {
+      alert('Please pick a style, size, and quantity');
+    } else {
+      console.log('atcqty', atcQuantity);
+      console.log('size', size);
```

Comment:
Mental note for future cleanup on console logs (not super important), shouldn't be done yet but before our final product is finished